client/reader/start/style.scss

```diff
@@ -81,14 +81,26 @@
 		font-family: $serif;
 		font-size: 12px;
 		margin: 10px 0 -3px;
-		max-height: none;
-	}
+		max-height: 65px;
 
-	.post-excerpt.is-long {
+		@include breakpoint( ">480px" ) {
+			max-height: 56px;
+		}
 
 		&:after {
-			display: none;
+			@include long-content-fade;
+			height: 25px;
+			position: absolute;
+				bottom: 0;
+				top: 40px;
+			width: 65%;
+
+			@include breakpoint( ">680px" ) {
```

Comment:
The linter is complaining about this width:

```WARNING: ERROR in breakpoint( >680px ): You can only use these sizes[  480px 660px 960px 1040px ] using the following syntax [ <480px >480px 480px-660px ]
	on line 48 of assets/stylesheets/shared/mixins/_breakpoints.scss
	from line 98 of client/reader/start/style.scss```

Is it alright to change it to 660px?